cleancode.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+echo "---------- Running ESLint ----------"
+npx eslint --fix frontend/.
+echo "------------------------------------
+"
```

Comment:
Any reason for the end quote `"` to be here instead on line 6?

Reply:
It's one way of printing a " \n " in bash. We can find a better way if it's not the best way to do it

Reply:
We can also use `echo -e "------------------------------------\n\n"`, which one should we use it?

Reply:
I'd say the latter is more clear. This way it doesn't look like a mistake. ☺️

Reply:
One way of adding a new line is by simply calling the `echo` command without any arguments